less/shared.less

```diff
@@ -263,10 +287,7 @@ input[type="email"] {
 
 input[type="email"] {
   padding-right: 33px;
-}
-input[name="lastname"],
-input[name="zip"] {
-  padding-left: 14px;
+  margin-top: 3px;
```

Comment:
Right, not related to RTL, fixing this on Chrome and Safari (Note the cut gradient at the top):
<img width="411" alt="capture d ecran 2017-01-24 a 01 41 20" src="https://cloud.githubusercontent.com/assets/1294206/22230591/5e14bb4a-e1d6-11e6-8a97-061172e7bbdd.png">